### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by loudmouth**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#### Added
+- Support for installation via Carthage on all of iOS, macOS, tvOS, & watchOS.
+
+---
+
+## [`0.3.0`](https://github.com/contentful/contentful.swift/releases/tag/0.3.0)
+Released 2017-01-08.
+
+#### Added
+- **Breaking: Swift 3, Xcode 8**
```
Comment (realityking) [inline CHANGELOG.md]: IMHO not a useful comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#### Changed
+- 
+
+#### Removed
+- 
+
+#### Fixed
+- 
+
+#### Deprecated
```
Comment (realityking) [inline CHANGELOG.md]: Up to you but personally I only ad these sections the first time I need to use them for a given release. I don't trust myself to remember to remove them 😅

File: CHANGELOG.md (hunk 1 above)
Comment (realityking) [inline CHANGELOG.md]: IMHO not a useful comment. I'd phrase it like `Converted to use Swift 3 and Xcode 8. Older versions are not supported.`

Comment (loudmouth) [inline CHANGELOG.md]: 👍  Nice I like the way you phrased it :-)

File: CHANGELOG.md (hunk 2 above)
Comment (loudmouth) [inline CHANGELOG.md]: Yeah I think you're right

**Pull request merged**